README.md

```diff
@@ -683,6 +683,7 @@ Here are some companies/projects (alphabetical order) using OpenAPI Generator in
 - [YITU Technology](https://www.yitutech.com/)
 - [Yelp](https://www.yelp.com/)
 - [Zalando](https://www.zalando.com)
+- [Bunker Holding Group](https://www.bunker-holding.com/)
```

Comment:
thanks for the PR. I'll merge it and rearrange it in alphabetical order via another PR.

Reply:
Great, thanks a lot.